### ISSUE TITLE
Topic in latest ArXiV documents at 2022-03-01 20:15:56

### ISSUE THREAD

**Issue by wanted2** [wanted2/caineng.in#520]
**Is your feature request related to a problem? Please describe.**
Topic in latest ArXiV documents at 2022-03-01 20:15:56
**Describe the solution you'd like**
* Topic 0: 
  * _Patterns of near-crash events in a naturalistic driving dataset:  applying rules mining_
  * _Implementing simple spectral denoising for environmental audio  recordings_
  * _Parameterized Intractability for Multi-Winner Election under the  Chamberlin-Courant Rule and the Monroe Rule_
  * _Iterated Belief Change, Computationally_
  * _Formulating Beurling LASSO for Source Separation via Proximal Gradient  Iteration_
  * _Rethinking Immersive Virtual Reality and Empathy_
  * _UnifiedQA-v2: Stronger Generalization via Broader Cross-Format Training_
  * _On change of measure inequalities for $f$-divergences_
  * _Program Synthesis for the OEIS_
  * _Measuring Trustworthiness or Automating Physiognomy? A Comment on Safra,  Chevallier, Grèzes, and Baumard (2020)_
* Topic 1: 
  * _Audio Visual Scene-Aware Dialog Generation with Transformer-based Video  Representations_
  * _Reproducibility in Optimization: Theoretical Framework and Limits_
  * _Minimax in Geodesic Metric Spaces: Sion's Theorem and Algorithms_
  * _Long-Time Convergence and Propagation of Chaos for Nonlinear MCMC_
  * _Single Trajectory Nonparametric Learning of Nonlinear Dynamics_
  * _Differentially Private Regression with Unbounded Covariates_
  * _Finding Safe Zones of policies Markov Decision Processes_
  * _Generalized Bayesian Additive Regression Trees Models: Beyond  Conditional Conjugacy_
  * _Reachability analysis in stochastic directed graphs by reinforcement  learning_
* Topic 2: 
  * _BeautifAI -- A Personalised Occasion-oriented Makeup Recommendation  System_
  * _Unintended Bias in Language Model-driven Conversational Recommendation_
  * _On the Overlooked Significance of Underutilized Contextual Features in  Recent News Recommendation Models_
  * _Movie Recommender System using critic consensus_
  * _QuoteR: A Benchmark of Quote Recommendation for Writing_
  * _Towards Fair Recommendation in Two-Sided Platforms_
  * _CSSR: A Context-Aware Sequential Software Service Recommendation Model_
  * _Sequential Item Recommendation in the MOBA Game Dota 2_
  * _Knowledge graph enhanced recommender system_
* Topic 3: 
  * _Deep Bayesian ICP Covariance Estimation_
  * _Intelligent Online Selling Point Extraction for E-Commerce  Recommendation_
  * _Active Learning for Point Cloud Semantic Segmentation via  Spatial-Structural Diversity Reasoning_
  * _3D Point Cloud Reconstruction and SLAM as an Input_
  * _PointSCNet: Point Cloud Structure and Correlation Learning Based on  Space Filling Curve-Guided Sampling_
  * _CG-SSD: Corner Guided Single Stage 3D Object Detection from LiDAR Point  Cloud_
  * _Point Cloud Denoising via Momentum Ascent in Gradient Fields_
  * _Variable Rate Compression for Raw 3D Point Clouds_
  * _On the impact of VR assessment on the Quality of Experience of Highly  Realistic Digital Humans_
  * _PointPCA: Point Cloud Objective Quality Assessment Using PCA-Based  Descriptors_
* Topic 4: 
  * _Punctuation Restoration_
  * _Improving Content-Aware Video Streaming in Congested Networks with  In-Network Computing_
  * _FoV Privacy-aware VR Streaming_
  * _GenéLive! Generating Rhythm Actions in Love Live!_
  * _DCNGAN: A Deformable Convolutional-Based GAN with QP Adaptation for  Perceptual Quality Enhancement of Compressed Video_
  * _MMSys'22 Grand Challenge on AI-based Video Production for Soccer_
  * _Nebula: Reliable Low-latency Video Transmission for Mobile Cloud Gaming_
  * _Banding vs. Quality: Perceptual Impact and Objective Assessment_
  * _A Multi-user Oriented Live Free-viewpoint Video Streaming System Based  On View Interpolation_
* Topic 5: 
  * _repro_eval: A Python Interface to Reproducibility Measures of  System-oriented IR Experiments_
  * _Establishing Strong Baselines for TripClick Health Retrieval_
  * _Atomized Search Length: Beyond User Models_
  * _Grep-BiasIR: A Dataset for Investigating Gender Representation-Bias in  Information Retrieval Results_
  * _HC4: A New Suite of Test Collections for Ad Hoc CLIR_
  * _Probably Reasonable Search in eDiscovery_
  * _Googling for Abortion: Search Engine Mediation of Abortion Accessibility  in the United States_
  * _Targeted Trojan-Horse Attacks on Language-based Image Retrieval_
  * _Customising Ranking Models for Enterprise Search on Bilingual  Click-Through Dataset_
  * _Search Strategy Formulation for Systematic Reviews: issues, challenges  and opportunities_
* Topic 6: 
  * _MELONS: generating melody with long-term structure using transformers  and structure graph_
  * _Armor: A Benchmark for Meta-evaluation of Artificial Music_
  * _Malakai: Music That Adapts to the Shape of Emotions_
  * _The HaMSE Ontology: Using Semantic Technologies to support Music  Representation Interoperability and Musicological Analysis_
  * _AccoMontage: Accompaniment Arrangement via Phrase Selection and Style  Transfer_
  * _Score Transformer: Generating Musical Score from Note-level  Representation_
  * _ChMusic: A Traditional Chinese Music Dataset for Evaluation of  Instrument Recognition_
  * _InverseMV: Composing Piano Scores with a Convolutional Video-Music  Transformer_
  * _Music2Video: Automatic Generation of Music Video with fusion of audio  and text_
  * _Deep Performer: Score-to-Audio Music Performance Synthesis_
* Topic 7: 
  * _Spectral embedding and the latent geometry of multipartite networks_
  * _An Efficient Pruning Process with Locality Aware Exploration and Dynamic  Graph Editing for Subgraph Matching_
  * _Anchor Graph Structure Fusion Hashing for Cross-Modal Similarity Search_
  * _(2.5+1)D Spatio-Temporal Scene Graphs for Video Question Answering_
* Topic 8: 
  * _Musical Audio Similarity with Self-supervised Convolutional Neural  Networks_
  * _A Smoothing and Thresholding Image Segmentation Framework with Weighted  Anisotropic-Isotropic Total Variation_
  * _Automated Reasoning in Non-classical Logics in the TPTP World_
  * _A Survey on Patients Privacy Protection with Stganography and Visual  Encryption_
  * _STEREO: Scientific Text Reuse in Open Access Publications_
* Topic 9: 
  * _DL-SLOT: Dynamic Lidar SLAM and Object Tracking Based On Graph  Optimization_
  * _Approximation error of single hidden layer neural networks with fixed  weights_
  * _Malaria detection in Segmented Blood Cell using Convolutional Neural  Networks and Canny Edge Detection_
  * _Privacy Leakage of Adversarial Training Models in Federated Learning  Systems_
  * _Real-time Over-the-air Adversarial Perturbations for Digital  Communications using Deep Neural Networks_
  * _Bit-wise Training of Neural Network Weights_
  * _Fostering the Robustness of White-Box Deep Neural Network Watermarks by  Neuron Alignment_
  * _Benign Overfitting in Two-layer Convolutional Neural Networks_
* Topic 10: 
  * _Relational Algebra and Calculus with SQL Null Values_
  * _SMILE: Sequence-to-Sequence Domain Adaption with Minimizing Latent  Entropy for Text Image Recognition_
  * _Graph-Relational Domain Adaptation_
  * _Cost-effective Framework for Gradual Domain Adaptation with  Multifidelity_
  * _Priming Cross-Session Motor Imagery Classification with A Universal Deep  Domain Adaptation Framework_
  * _Few-shot Unsupervised Domain Adaptation with Image-to-class Sparse  Similarity Encoding_
  * _BioADAPT-MRC: Adversarial Learning-based Domain Adaptation Improves  Biomedical Machine Reading Comprehension Task_
  * _Attention-based Cross-Layer Domain Alignment for Unsupervised Domain  Adaptation_
  * _Conditional Extreme Value Theory for Open Set Video Domain Adaptation_
* Topic 11: 
  * _Multimodal Incremental Transformer with Visual Grounding for Visual  Dialogue Generation_
  * _Local Quadruple Pattern: A Novel Descriptor for Facial Image Recognition  and Retrieval_
  * _Cascaded Asymmetric Local Pattern: A Novel Descriptor for Unconstrained  Facial Image Recognition and Retrieval_
  * _Improving Fake News Detection by Using an Entity-enhanced Framework to  Fuse Diverse Multimodal Clues_
  * _Local Gradient Hexa Pattern: A Descriptor for Face Recognition and  Retrieval_
  * _Centre Symmetric Quadruple Pattern: A Novel Descriptor for Facial Image  Recognition and Retrieval_
* Topic 12: 
  * _Causal discovery for observational sciences using supervised machine  learning_
  * _Causal Analysis Framework for Recommendation_
  * _Stochastic Causal Programming for Bounding Treatment Effects_
  * _Learning Latent Causal Dynamics_
  * _Benign-Overfitting in Conditional Average Treatment Effect Prediction  with Linear Regression_
  * _The Causal Marginal Polytope for Bounding Treatment Effects_
  * _Effect Identification in Cluster Causal Diagrams_
  * _Deep Causal Reasoning for Recommendations_
  * _Diffusion Causal Models for Counterfactual Estimation_
* Topic 13: 
  * _CAISE: Conversational Agent for Image Search and Editing_
  * _SketchEdit: Mask-Free Local Image Manipulation with Partial Sketches_
  * _Data-Consistent Local Superresolution for Medical Imaging_
  * _Seamless Copy Move Manipulation in Digital Images_
  * _ciscNet -- A Single-Branch Cell Instance Segmentation and Classification  Network_
  * _Perceptual Consistency in Video Segmentation_
  * _Confidence Calibration for Object Detection and Segmentation_
* Topic 14: 
  * _Multi-Cue Adaptive Emotion Recognition Network_
* Topic 15: 
  * _Clustering by Hill-Climbing: Consistency Results_
  * _A sampling-based approach for efficient clustering in large datasets_
  * _Strong Consistency for a Class of Adaptive Clustering Procedures_
  * _Web-Based File Clustering and Indexing for Mindoro State University_
  * _Use Image Clustering to Facilitate Technology Assisted Review_
  * _Optimal Clustering with Bandit Feedback_
  * _Mind Your Clever Neighbours: Unsupervised Person Re-identification via  Adaptive Clustering Relationship Modeling_
  * _Clustering by the Probability Distributions from Extreme Value Theory_
  * _Human Motion Detection Using Sharpened Dimensionality Reduction and  Clustering_
* Topic 16: 
  * _Disentangling Hate in Online Memes_
  * _Evaluation of an Audio-Video Multimodal Deepfake Dataset using Unimodal  and Multimodal Detectors_
* Topic 17: 
  * _Interpolation and Regularization for Causal Learning_
  * _Learning Causal Overhypotheses through Exploration in Children and  Computational Models_
  * _Relational Causal Models with Cycles:Representation and Reasoning_
* Topic 18: 
* Topic 19: 
  * _Large-scale Stochastic Optimization of NDCG Surrogates for Deep Learning  with Provable Convergence_
  * _Rectified Max-Value Entropy Search for Bayesian Optimization_
* Topic 20: 
  * _A Robust Multi-Objective Bayesian Optimization Framework Considering  Input Uncertainty_
* Topic 21: 
  * _Local Directional Gradient Pattern: A Local Descriptor for Face  Recognition_
* Topic 22: 
  * _Enabling arbitrary translation objectives with Adaptive Tree Search_
  * _Robust Training under Label Noise by Over-parameterization_
  * _Diagnosing BERT with Retrieval Heuristics_
* Topic 23: 
  * _A PDE-Based Analysis of the Symmetric Two-Armed Bernoulli Bandit_
  * _Meta-Learning for Simple Regret Minimization_
  * _LSTM-RPA: A Simple but Effective Long Sequence Prediction Algorithm for  Music Popularity Prediction_
  * _Efficient and Differentiable Conformal Prediction with General Function  Classes_
* Topic 24: 
  * _Provable Stochastic Optimization for Global Contrastive Learning: Small  Batch Does Not Harm Performance_
* Topic 25: 
  * _Memorize to Generalize: on the Necessity of Interpolation in High  Dimensional Linear Regression_
  * _Evaluating the Adversarial Robustness of Adaptive Test-time Defenses_
  * _Improving Robustness of Convolutional Neural Networks Using Element-Wise  Activation Scaling_
  * _Bidding Agent Design in the LinkedIn Ad Marketplace_
* Topic 26: 
* Topic 27: 
  * _FNR: A Similarity and Transformer-Based Approach to Detect Multi-Modal  Fake News in Social Media_
* Topic 28: 
  * _A Real-World Implementation of Unbiased Lift-based Bidding System_
* Topic 29: 
  * _PQuAD: A Persian Question Answering Dataset_
* Topic 30: 
* Topic 31: 
  * _Why Do We Click: Visual Impression-aware News Recommendation_
* Topic 32: 
  * _Phase Continuity: Learning Derivatives of Phase Spectrum for Speech  Enhancement_
  * _Speaker Adaptation Using Spectro-Temporal Deep Features for Dysarthric  and Elderly Speech Recognition_
* Topic 33: 
  * _Large-scale Personalized Video Game Recommendation via Social-aware  Contextualized Graph Neural Network_
  * _Learning to Identify Perceptual Bugs in 3D Video Games_
  * _How player and opponent personalities influence cooperative gameplay_
  * _Player Dominance Adjustment in Games_
  * _Fighting Game Commentator with Pitch and Loudness Adjustment Utilizing  Highlight Cues_
* Topic 34: 
  * _FUNQUE: Fusion of Unified Quality Evaluators_
* Topic 35: 
  * _An End-to-End Transformer Model for Crowd Localization_
  * _FusionCount: Efficient Crowd Counting via Multiscale Feature Fusion_
  * _Wisdom of (Binned) Crowds: A Bayesian Stratification Paradigm for Crowd  Counting_
  * _Reinforcing Local Feature Representation for Weakly-Supervised Dense  Crowd Counting_
  * _Multiscale Crowd Counting and Localization By Multitask Point  Supervision_
* Topic 36: 
  * _Integrating psychotherapy practices and gamified elements in novel game  mechanics for stress relief_
  * _Deconstructing Distributions: A Pointwise Framework of Learning_
* Topic 37: 
  * _Cross-Task Knowledge Distillation in Multi-Task Recommendation_
  * _TURNER: The Uncertainty-based Retrieval Framework for Chinese NER_
  * _A Complete Criterion for Value of Information in Soluble Influence  Diagrams_
* Topic 38: 
  * _MAVE: A Product Dataset for Multi-source Attribute Value Extraction_
* Topic 39: 
  * _Towards Reducing Manual Workload in Technology-Assisted Reviews:  Estimating Ranking Performance_
* Topic 40: 
* Topic 41: 
  * _Nonlinear Discrete Optimisation of Reversible Steganographic Coding_
* Topic 42: 
  * _Robotic Telekinesis: Learning a Robotic Hand Imitator by Watching Humans  on Youtube_
  * _Comfort and Sickness while Virtually Aboard an Autonomous Telepresence  Robot_
* Topic 43: 
* Topic 44: 
  * _On PAC-Bayesian reconstruction guarantees for VAEs_
  * _Data Overlap: A Prerequisite For Disentanglement_
* Topic 45: 
  * _Seeing Like a Toolkit: How Toolkits Envision the Work of AI Ethics_
* Topic 46: 
  * _Deepfake Detection for Facial Images with Facemasks_
* Topic 47: 
* Topic 48: 
* Topic 49: 
  * _CoSeg: Cognitively Inspired Unsupervised Generic Event Segmentation_
  * _Bina-Rep Event Frames: a Simple and Effective Representation for  Event-based cameras_

**Describe alternatives you've considered**
None
**Additional context**
None
